otx/cli/builder/builder.py

```diff
@@ -140,9 +140,9 @@ class Builder:
     def build_task_config(
         self,
         task_type: str,
+        workspace_root: str,
         model_type: str = None,
         train_type: str = "incremental",
-        workspace_path: Union[Path, str] = None,
```

Comment:
As far as I remember, in unit tests, this is passed as a `Path`, but wouldn't this be a problem?

Reply:
If so, I'll change the unit-test soon !

Reply:
I edited unit-test also :)